CONTRIBUTING.md

```diff
@@ -51,6 +51,116 @@ You will get the values for the following environment from the Firebase project
 
 Open a terminal and run `npm run dev` to start the development server, and start using the local version of CPE.
 
+## Coding Rules
+
+This section provides guidelines for writing clean, consistent, and maintainable code, covering naming conventions, code structure, documentation, error handling, and performance. Adhering to these rules ensures high-quality, reliable software.
+
+### Writing Documentation
+
+Quality documentation is crucial for the success of any development project. It facilitates teamwork, maintenance, and product evolution. This section covers key principles and best practices for writing effective documentation.
+
+Since technology doesn't stand still these days, you can use this prompt for chatgpt to write documentation.
+
+```plaintext
+ChatGPT Documentation Prompt
+
+When documenting ChatGPT code, it's crucial to provide clear and comprehensive information about each function, method, and class. This ensures that the code is understandable and maintainable. Follow these guidelines to document your ChatGPT TypeScript code effectively:
+
+1. Function or Method Documentation:
+   - Description: Provide a brief but detailed description of what the function or method does.
+   - @param: For each parameter, describe the type and purpose.
+   - @returns: Describe the return value and its type, if applicable.
+   - @example: Include an example of how to use the function or method.
+
+2. Class Documentation:
```

Comment:
Hi @FussuChalice, thanks for preparing this PR. As of now, I plan not to use Class anymore for the new Organized app. However, while component and pages are being developed, it’s okay if you use it for mocking some data. Logics will be slightly updated later when integrating the functionality with the real data from IndexedDb. Thank you.